webproject/src/main/java/com/google/sps/data/Order.java

```diff
@@ -0,0 +1,60 @@
+package com.google.sps.data;
+
+public class Order {
+    private String user;
+    private String product;
+    private int quantity;
+    private double unitPrice;
```

Comment:
I think for price you should have a new data structure to hold the currency of the price

Reply:
Might be a bit difficult to include currency for now because we need to check the total order value of a room, maybe we can move it to after the presentation?

Reply:
good idea, I think we can keep the currency as an attribute in the room later. lets keep this for now